Glyssen/Bundle/GlyssenDblTextMetadata.cs

```diff
@@ -263,6 +268,27 @@ public QuoteSystemStatus QuoteSystemStatus
 		public VoiceActorStatus VoiceActorStatus { get; set; }
 	}
 
+	public class ProjectSettings
+	{
+		public const int kNumberOfNarratorsNotSet = 0;
+
+		/// <summary>
+		/// Number of narrators set by user
+		/// 0 if not set by user
+		/// </summary>
+		/// 
+		[XmlElement("numberofnarrators")]
```

Comment:
I did attributes at first (not noticing the difference). Andrew told me to change it to elements and described what he wanted, so I worked it up that way